modules/nextflow/src/main/groovy/nextflow/executor/AbstractGridExecutor.groovy

```diff
@@ -205,7 +206,7 @@ abstract class AbstractGridExecutor extends Executor {
     /**
      * Given the string returned the by grid submit command, extract the process handle i.e. the grid jobId
      */
-    abstract parseJobId( String text );
+    abstract String parseJobId( String text );
```

Comment:
It would be preferable to avoid changing method signatures. It may impact third party plugins